grpc/server/pom.xml

```diff
@@ -34,6 +34,7 @@
     <properties>
         <version.zipkin-junit>2.12.5</version.zipkin-junit>
         <version.lib.bedrock>5.0.11</version.lib.bedrock>
+        <version.netty.tcnative>2.0.20.Final</version.netty.tcnative>
```

Comment:
Version properties should only be in the very top level pom.xml. I realise that some of these were already here but they should all be removed.